text/0000-compiler-team-trusted-contrib-maintainer.md

```diff
@@ -0,0 +1,381 @@
+contributor will be moved to alumni status. Members who are no longer able to
+help maintain the compiler but otherwise wish to continue contributing to the
+compiler can also be moved to alumni status and retain their trusted contributor
+status. Alumni can ask to be reinstated in future.
+
+[coc]: https://www.rust-lang.org/policies/code-of-conduct
+
+Responsibilities
+----------------
+[responsibilities]: #responsibilities
+
+There are various responsibilities that a team member could take on to help the
+team. All team members must participate in at least one activity.
+
+Team members can get involved in any of these by contacting the team leads, by
+asking team members currently involved in these responsibilities, or by asking
+in any venue where these responsibilities are conducted (e.g. a Zulip stream).
+
+- Final comment period (FCP) reviewer
+    - Final comment periods are the process by which the team signs-off on a
+      change before it is made, like stabilizing a feature.
+
+      FCPs have always required whole team to sign-off, but this doesn't scale
+      as the team grows. As described above, this acts as a disincentive for the
+      team to grow. Furthermore, not all FCPs are relevant to all team members and a
+      diffusion of responsibility means that most team members just sanity-check and
+      then sign-off. This isn't ideal, as it doesn't guarantee that someone on the
+      team has thoroughly considered a FCP.
+
+      Instead, have FCPs require sign-off from team members who opt-in to being
+      an "FCP reviewer", with the expectation that they will spend time reviewing
+      an FCP thoroughly. FCP reviewers should also consider reaching out to relevant
+      domain experts and soliciting their opinions whenever possible. Any team or
+      project member can raise concerns with an FCP, which will be considered by the
+      FCP reviewers.
+      
+      To function effectively, it is recommended that there be 4 - 8 FCP
+      reviewers at any time. If less than 4 FCP reviewers are available, the compiler
+      team co-leads will act as FCP reviewers until the reviewers can be found.
+
+- Performance triage
+    - There is a rotation of team members and other project members who check
+      all of the interesting performance benchmarks from the last week to produce a
+      report summarizing the improvements and regressions. This is valuable to keep
+      track of the compiler's performance over time and make sure that regressions are
+      being addressed.
+
+- Issue prioritisation
+    - The compiler team has a prioritisation procedure and policy to identify
+      and label issues according to their importance. These labels feed into the
+      backport procedure (what's worth being backported) and work priorities of team
+      members.
+
+- Backport reviews
+    - Each week, some team members participate in the weekly triage meeting
```

Comment:
we've talked about this before but backport decisions should really be changed to be async, so I think it makes sense to be less precise about them on RFC and just say that this means "regularly" deciding on backports

Reply:
This makes sense, fixed!